llvm/docs/ReleaseNotes.md

```diff
@@ -460,6 +460,8 @@ Changes to the LLVM tools
   `--localize-symbol`, `--localize-symbols`,
   `--skip-symbol`, `--skip-symbols`.
 
+* llvm-objcopy now prints the correct file path in the error message when the output file specified by --dump-section cannot be opened.
```

Comment:
double quotes around the option name.

Reply:
I made it `--dump-section` to be similar to all other options in the same file